dev-tools/compose/docker-compose.test.yml

```diff
@@ -241,6 +241,9 @@ services:
       audius_elasticsearch_url: 'http://discovery-provider-elasticsearch:9200'
     volumes:
       - ${PROJECT_ROOT}/packages/discovery-provider:/audius-discovery-provider
+      - ${PROJECT_ROOT}/packages/es-indexer:/app/packages/discovery-provider/es-indexer
+      - ${PROJECT_ROOT}/node_modules:/app/node_modules
```

Comment:
wow local modules work? sometimes they don't b/c of arch

Reply:
oh interesting, yeah it worked in this case for running the tests